### DISCUSSION TITLE
Account frozen

### DISCUSSION THREAD

**Discussion by SinoMotors** [vercel/community#6617] [Account Issue]
### Question

Good afternoon.
We had a Pro account "sgg365-gmailcom". Switched to a different platform. Now we can't disable and delete the account in your platform. And the card is not charged for the overdue account, and we can't do anything with the account because Invoice number 9E2843AC-0006
Date of issue March 19, 2024 not paid.
I want to delete the account in your platform. Please help me to resolve this global issue.

**Comment by amyegan** [comment]
Hey @SinoMotors. Please [submit a case](https://vercel.com/help) to the Customer Success Team. You should choose `Invoice Enquiry` as the **Problem Area** when submitting the case. That will get you in touch with the our billing experts who can help you get this sorted
